packages/components/Checkbox/src/Checkbox.settings.ios.ts

```diff
@@ -0,0 +1 @@
+export * from './Checkbox.settings.android';
```

Comment:
I don't think this is correct.
Checkbox.settings.windows.ts was created because windows/win32 doesn't support SVG's. It looks like you renamed settings.windows to settings.android and are using it for iOS/macOS. That means the settings for non-SVG supporting platforms are being used for SVG supporting platforms. 

Also, I'd rather not have one platforms settings be imported into another platforms. I'd rather have some common name like "settings / settingsWithoutSVG" that each imports if that's the actual distinction.

Reply:
I didn't rename the .windows one to .android. I took the base Checkbox.settings.ts from before, and made that into Checkbox.settings.android.ts. I then made the base Checkbox.setting.ts into the _normal_ Win32/Windows .settings file from before [this PR](https://github.com/microsoft/fluentui-react-native/pull/716/)

It's all a bit confusing, but let me try to simplify it some more.

Checkbox.settings.ts should have never been changed to be the default settings files for android/iOS/macOS/Web (which it was in the previous PR). Checkbox.setting.ts should be the default file for Win32/Windows, and the other platforms (if needed) should have separate platform-specific files. 

Also, I understand your sentiment of not having one platform import from another platform. However, if we do combine the Android/MacOS/iOS into one file named something like _Checkbox.settingsWithSVG_, what happens if there comes a time where we actually need to make a change to only iOS and not Android?

Reply:
I see, sorry I got confused looking at what git _thought_ you did in the headers for the file changes.
Come that time, I assume we would either do a `if Platform === 'ios'` or `Platform.Select({...})` check, or separate out the types even further. 
This is probably valid for talking about in person since this and the other thread are related =)

Reply:
No problem, I just noticed that and it's definitely confusing :)